The rx_count and rx_list attributes were only added in kernel 5.4 in
commit 323ebb61e32b ("net: use listified RX for handling GRO_NORMAL skbs")

--- a/drivers/net/wireless/intel/iwlwifi/pcie/rx.c
+++ b/drivers/net/wireless/intel/iwlwifi/pcie/rx.c
@@ -1539,11 +1539,13 @@ out:
 	if (napi->poll) {
 		napi_gro_flush(napi, false);
 
+#if LINUX_VERSION_IS_GEQ(5,4,0)
 		if (napi->rx_count) {
 			netif_receive_skb_list(&napi->rx_list);
 			INIT_LIST_HEAD(&napi->rx_list);
 			napi->rx_count = 0;
 		}
+#endif
 	}
 
 	iwl_pcie_rxq_restock(trans, rxq);
